networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -26,4 +26,18 @@ The Network Observability Operator is now stable and the release channel is upgr
 === Bug fix
 
 * Previously, unless the Loki `authToken` configuration was set to `FORWARD` mode, authentication was no longer enforced, allowing any user who could connect to the {product-title} console in an {product-title} cluster to retrieve flows without authentication.
-Now, regardless of the Loki `authToken` mode, only cluster administrators can retrieve flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2169468[*BZ#2169468*])
\ No newline at end of file
+Now, regardless of the Loki `authToken` mode, only cluster administrators can retrieve flows. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2169468[*BZ#2169468*])
+
+[id="network-observability-operator-release-notes-1-2"]
+== Network Observability Operator 1.2.0
+
+The following advisory is available for the Network Observability Operator 1.1.0:
+
+* <errata link here>
+
+[id="network-observability-operator-preparing-to-update"]
+=== Preparing for the next update 
+
+The subscription of an installed Operator specifies an update channel that tracks and receives updates for the Operator. Until the 1.2 release of the Network Observability Operator, the only channel available was `v1.0.x`. The 1.2 release of the Network Observability Operator introduces the `stable` update channel for tracking and receiving updates. You must switch your channel from the default `v1.0.x` to `stable` to recieve future Operator updates. The `v1.0.x` channel is planned to be deprecated in following releases.
+
+The names of update channels in a subscription can differ between Operators, but the naming scheme typically follows a common convention within a given Operator. For example, Network Observability Operator channel names might follow a minor release update stream for the application provided by the Operator, such as `v1.0.x` or a release frequency, such as `stable` or `fast`.
```

Comment:
@skrthomas Ngl, I'm not sure what the point of this section is...

Reply:
In Line 43, to be specific

Reply:
I guess its a little more background info on subscription channel names in general. I noticed it was here in the [compliance operator](https://docs.openshift.com/container-platform/4.12/security/compliance_operator/compliance-operator-updating.html) updating topic. I guess once we have only a single stream, this paragraph will be really pointless. But while we have two, I was thinking maybe some context would be helpful? Could go either way as TMI for the release note or helpful contextual info.

Reply:
I would also consider removing this part because it could add confusion to the readers, as they could think we are indeed doing multistream - but we don't. Starting with v1.0.x was a mistake so we introduce the correct channel now, but despite having two channels we still doing single stream. In multistream, the two channels would live their independent life, each receiving their own updates. We're not going to do that, `v1.0.x` will just be strictly equal to `stable` for a few releases before being definitively stopped.

Reply:
Okay, I'll remove it. I can see how the text could be misleading given starting with 1.0.x was a mistake. Addressed.